build/source/driver/multi_driver.f90

```diff
@@ -369,69 +525,88 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 do istep=1,numtim
 
  ! set print flag
- globalPrintFlag=.false.
+ globalPrintFlag=.true.
 
  ! read a line of forcing data (if not already opened, open file, and get to the correct place)
- ! NOTE: only read data once: if same data used for multiple HRUs, data is copied across
  do iHRU=1,nHRU  ! loop through HRUs
-  ! assign pointers to HRUs
-  time_data => time_hru(iHRU)
-  forc_data => forc_hru(iHRU)
-  ! read forcing data
-  call read_force(istep,iHRU,err,message); call handle_err(err,message)
+  if(forcFileInfo(iHRU)%ixFirstHRU > 0)then
```

Comment:
Yes, where the forcing data is used across multiple HRUs it is only read in once.

HOWEVER: This code is removed in the subsequent PR for upgrading to NetCDF input, so this code (and lack of commenting) is left unchanged).